src/formats/CML.cpp

```diff
@@ -50,29 +50,29 @@ CMLFormat::CMLFormat(std::string path, File::Mode mode, File::Compression compre
 
     auto result = document_.load(*file_);
     if (!result) {
-        throw format_error("[CML] Parsing error: '{}'", result.description());
```

Comment:
I like the idea of putting the same of the format in the error message like is done for `xzstreambuf`. This way users can know what file format an error is associated with when an error is thrown.

Reply:
I can make my own PR to add these to the messages and update the check script.

Reply:
I see what you mean. I kinda like the `... in <format>` or `... for <format>` approach, since it flows better. I could also live with `format: ...` as the message.

A possible way to ensure this is used everywhere would be to include a mandatory context to the warning function:

```cpp
void warning(std::string context, std::string message, Args...);

warning("cml format", "...", ..., ..., ...);
// or 
warning("cml reader", "...", ..., ..., ...);
warning("cml writer", "...", ..., ..., ...);

warning("pdb format", "...", ..., ..., ...);
```

Reply:
That would work as well!